### ISSUE TITLE
BLines/Lines doesn't respect vim's settings for whitespaces

### ISSUE THREAD

**Issue by synaptiko** [junegunn/fzf.vim#49]
I'm using following vim's settings:
```vim
set tabstop=2
set softtabstop=2
set shiftwidth=2
set noexpandtab

set listchars=tab:  ,trail:·,space:·,nbsp:·
```

But when I run `:BLines` or `:Lines` I can see that tab is expanded into 8 spaces which is pretty awful and it makes orientation among lines very hard for me.

**Comment by junegunn** [comment]
fzf as being an external command-line program has its own rendering engine where `tabstop` of vim does not apply. Currently fzf uses tabstop of 8 and has no option to override it. Adding the option (e.g. `fzf --tabstop=2`) to fzf is not impossible but, `tabstop` setting of Vim is local to each buffer and since `:Lines` will output lines from all open buffers that may have heterogeneous `tabstop` config, it does not suffice. Which means we have to preprocess tab characters while preparing input to fzf, but it can slow down the processing due to the slow performance and synchronous nature of vimscript.

**Comment by synaptiko** [comment]
Ok, I understand and it's unfortunate, I didn't realize that each buffer has own settings.

Although I think that for most cases it would be enough to have `--tabstop` option because then you could add it to `FZF_DEFAULT_OPTS`.

**Comment by junegunn** [comment]
I'll consider that.

**Comment by synaptiko** [comment]
Thank you... it seems you've already implemented it :+1:

When are you going to release it?

**Comment by junegunn** [comment]
Just released 0.11.1 and applied the new option to the command (also `--reverse`). Thanks for the suggestion :)
